### DISCUSSION TITLE
Snippet Ordering & Sorting

### DISCUSSION THREAD

**Discussion by HANZ64** [massCodeIO/massCode#64] [Ideas]
Hey there,

Currently the program orders all snippets based on when they were last updated (newest updates starting from the top).

I would like to suggest a feature where we can simply drag and drop snippets in any order we want (including favorites).

Perhaps there could also be an option to sort by last updated, date created, alphabetical etc.

**Comment by antonreshetov** [comment]
@HANZ64 sort by last updated, date created, alphabetical is already in the backlog. So it will appear in future releases.

**Reply by antonreshetov** [reply]
Implemented in #69

**Reply by HANZ64** [reply]
> Implemented in #69

Just tested out the new build, and it works really well!

One small addition I would request is to be able to sort by date created but in reverse. Right now the oldest items are all the way to the bottom, and the newest items are at the top of the list (my preference is to have it the other way).

That is actually the main reason why I requested this feature, I simply want to order my snippets by the date I created them with the oldest items starting from the top. Ideally it would be great to manually position snippets in any order we like, but I understand that will require more coding work to implement.

If you do implement this, I'd suggest adding reverse options for date modified and name also.

Thank you!
